src/Map.jsx

```diff
@@ -103,7 +104,37 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  /**
+   * return coordinates in an array [lon, lat]
+   */
+  getCoordinateArray(stop) {
+    console.log(this);
+    const stopPoints = [];
+    stopPoints.push(stop.lat);
+    stopPoints.push(stop.lon);
+    return stopPoints;
+  }
```

Comment:
could we do `return [stop.lat, stop.lon];` instead? If so, then I feel that this function won't be necessary. (as `stop =>( [stop.lat, stop.lon])`) is cleaner)